kasenvr/project-athena
# Improving performance of entity updates

This is a fairly risky PR that contains various fixes to try to improve the performance of entity updates.  This is the first step in a project to improve parenting to fix things like the Shape 3D app and vehicles.

To test:
- Have two people running this PR go to a server running this PR.
- Have one person create and modify entities.  Have them change as many properties as possible that are relevant to rendering.  For example, have them toggle the visibility of a Cube Entity.  Have them change the color of a Text Entity.  Have them change an Image Entity's image.
- For each change, both observers should see the change take effect.
